### DISCUSSION TITLE
Import Versions - between Mobx & MobxReact

### DISCUSSION THREAD

**Discussion by pixman92** [mobxjs/mobx#3623] [Q&A]
I am using Codepen and import statements, but my imports
```
import React, { useState } from "https://esm.sh/react@18.2.0";
import ReactDOM from "https://esm.sh/react-dom@18.2.0";
import * as reactRouter from "https://cdn.skypack.dev/react-router@6.6.2";
import * as mobxReact from "https://cdn.skypack.dev/mobx-react@7.6.0";
import {
  makeObservable,
  observable,
  computed,
  action,
  autorun
} from "https://cdn.skypack.dev/mobx@6.7.0";
```
cause this error:

`Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7B%24%24typeof%2C%20type%2C%20compare%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.`

What's going wrong?

**Comment by mweststrate** [comment]
That is a react error, not a MobX error, and without repo we can look at. So little we can do for you here, but I suggest you keep removing pieces from your application until it works, given the error it seems you are passing children to a react component in a wrong way.

**Reply by pixman92** [reply]
Oh thanks for the insight!

the Codpen.io repo is:
https://codepen.io/christianbear1221/pen/PoBEerY
